Docker build fails with msgs.Messages undefined (type []*vici.Message has no field or method Messages)

Hi, 

Docker build fails with `msgs.Messages undefined`.

```
22.06 ./main.go:30:24: msgs.Messages undefined (type []*vici.Message has no field or method Messages)
------
Dockerfile:9
--------------------
   7 |     #	go mod download
   8 |     COPY app /app
   9 | >>> RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main .
  10 |
  11 |     FROM alpine:latest
--------------------
ERROR: failed to solve: process "/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main ." did not complete successfully: exit code: 2
```

--- a/app/main.go
+++ b/app/main.go
@@ -45,8 +45,5 @@ func listSAs() ([]LoadedIKE, error) {
 		}
 	}
 
-	// close the socket
-	s.Close()
-
 	return retVar, nil
 }

--- a/app/main.go
+++ b/app/main.go
@@ -14,35 +14,34 @@ func main() {
 	http.Handle("/metrics", promhttp.Handler())
 	log.Fatalln(http.ListenAndServe(":9814", nil))
 }
-func listSAs()([]LoadedIKE, error){
-	s, err := vici.NewSession()
-	if err != nil {
-		log.Printf("Error Connecting to vici: %s", err)
-		return nil, err
-	}
-	defer s.Close()
+func listSAs() ([]LoadedIKE, error) {
+    s, err := vici.NewSession()
+    if err != nil {
+        log.Printf("Error Connecting to vici: %s", err)
+        return nil, err
+    }
+    defer s.Close()
 
-	var retVar []LoadedIKE
-	msgs, err := s.StreamedCommandRequest("list-sas", "list-sa", nil)
-	if err != nil {
-		return retVar, err
-	}
-	for _,m := range msgs.Messages() {
-		if e := m.Err(); e != nil{
-			//ignoring this error
-			continue
-		}
-		for _, k := range m.Keys() {
-			inbound := m.Get(k).(*vici.Message)
-			var ike LoadedIKE
-			if e := vici.UnmarshalMessage(inbound, &ike); e != nil {
-				//ignoring this marshal/unmarshal errro!
-				continue
-			}
-			ike.Name = k
-			retVar = append(retVar, ike)
-
-		}
-	}
-	return retVar, nil
+    var retVar []LoadedIKE
+    msgs, err := s.StreamedCommandRequest("list-sas", "list-sa", nil)
+    if err != nil {
+        return retVar, err
+    }
+    for _, m := range msgs { // <- Directly iterate over msgs
+        if e := m.Err(); e != nil {
+            //ignoring this error
+            continue
+        }
+        for _, k := range m.Keys() {
+            inbound := m.Get(k).(*vici.Message)
+            var ike LoadedIKE
+            if e := vici.UnmarshalMessage(inbound, &ike); e != nil {
+                //ignoring this marshal/unmarshal error!
+                continue
+            }
+            ike.Name = k
+            retVar = append(retVar, ike)
+        }
+    }
+    return retVar, nil
 }

--- a/app/main.go
+++ b/app/main.go
@@ -1,10 +1,11 @@
 package main
 
-import(
+import (
 	"log"
-	"github.com/strongswan/govici/vici"
 	"net/http"
+
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/strongswan/govici/vici"
 )
 
 func main() {
@@ -15,33 +16,37 @@ func main() {
 	log.Fatalln(http.ListenAndServe(":9814", nil))
 }
 func listSAs() ([]LoadedIKE, error) {
-    s, err := vici.NewSession()
-    if err != nil {
-        log.Printf("Error Connecting to vici: %s", err)
-        return nil, err
-    }
-    defer s.Close()
+	s, err := vici.NewSession()
+	if err != nil {
+		log.Printf("Error Connecting to vici: %s", err)
+		return nil, err
+	}
+	defer s.Close()
 
-    var retVar []LoadedIKE
-    msgs, err := s.StreamedCommandRequest("list-sas", "list-sa", nil)
-    if err != nil {
-        return retVar, err
-    }
-    for _, m := range msgs { // <- Directly iterate over msgs
-        if e := m.Err(); e != nil {
-            //ignoring this error
-            continue
-        }
-        for _, k := range m.Keys() {
-            inbound := m.Get(k).(*vici.Message)
-            var ike LoadedIKE
-            if e := vici.UnmarshalMessage(inbound, &ike); e != nil {
-                //ignoring this marshal/unmarshal error!
-                continue
-            }
-            ike.Name = k
-            retVar = append(retVar, ike)
-        }
-    }
-    return retVar, nil
+	var retVar []LoadedIKE
+	msgs, err := s.StreamedCommandRequest("list-sas", "list-sa", nil)
+	if err != nil {
+		return retVar, err
+	}
+	for _, m := range msgs { // <- Directly iterate over msgs
+		if e := m.Err(); e != nil {
+			//ignoring this error
+			continue
+		}
+		for _, k := range m.Keys() {
+			inbound := m.Get(k).(*vici.Message)
+			var ike LoadedIKE
+			if e := vici.UnmarshalMessage(inbound, &ike); e != nil {
+				//ignoring this marshal/unmarshal error!
+				continue
+			}
+			ike.Name = k
+			retVar = append(retVar, ike)
+		}
+	}
+
+	// close the socket
+	s.Close()
+
+	return retVar, nil
 }

